courses/models.py

```diff
@@ -151,7 +151,7 @@ class CourseRun(models.Model):
       rather a specific instance of that course being taught.
     """
     title = models.CharField(max_length=255)
-    edx_course_key = models.CharField(max_length=255, blank=True, null=True, db_index=True)
+    edx_course_key = models.CharField(max_length=255, blank=True, null=True, unique=True, )
```

Comment:
`unique=True` automatically implies an indexed field, so `db_index=True` is redundant (https://docs.djangoproject.com/en/1.10/ref/models/fields/#unique)